code/src/reducers/tasks.js

```diff
@@ -0,0 +1,62 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  items: [
+    {
+      id: 1,
+      description: 'Test 1',
+      priority: 'High',
+      completed: false
+    },
+    {
+      id: 2,
+      description: 'Test 2',
+      priority: 'Low',
+      completed: false
+    },
+    {
+      id: 3,
+      description: 'Test 3',
+      priority: '',
+      completed: false
+    }
+  ]
+}
+
+export const tasks = createSlice({
+  name: 'tasks',
+  initialState,
+  reducers: {
+    addItem: (state, action) => {
+      const newItem = {
+        id: Date.now(),
```

Comment:
It's bit inconsistent to initialise 3 items from initialState with ids as numbers, and later on add new items with ids as dates